Cargo.toml

```diff
@@ -25,3 +25,7 @@ num-integer = "0.1.42"
 num-traits = "0.2.11"
 num-derive = "0.3.0"
 hex = { version = "=0.3.2" }
+torut = { version = "0.1.1", optional = true }
```

Comment:
I would consider specifying features manually since by default it also compiles the controller implementation. I believe `["v2", "v3"]` should be enough: https://github.com/teawithsand/torut/blob/master/Cargo.toml#L17-L18

Reply:
good point!